habitat-mas/habitat_mas/agents/actions/base_actions.py

```diff
@@ -31,23 +31,57 @@ def send_request(request: str, target_agent: str) -> str:
     """
     pass
 
-# TODO(YCC): add description for this action
+@action
+def wait():
+    """
+    Wait if no immediate action is required from you.
+    """
+    pass
+
+@action
+def open_fridge():
+    """
+    Open the fridge. Only when you are near the fridge.
+    """
+    pass
+
+@action
+def close_fridge():
+    """
+    Close the fridge. Only when you are near the fridge.
+    """
+    pass
+
+@action
+def open_cab():
+    """
+    Open the cabinet. Only when you are near the cabinet.
+    """
+    pass
+
+@action
+def close_cab():
+    """
+    Close the cabinet. Only when you are near the cabinet.
+    """
+    pass
+
```

Comment:
Why do we want to add these actions to the agent skill library? I think we have not implemented these skills yet, no?

Reply:
Yep, we'd better remove them from the `ACTION_POOL`.